.storybook/addons.js

```diff
@@ -0,0 +1,4 @@
+import '@storybook/addon-actions/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-notes/register';
+import '@storybook/addon-viewport/register';
```

Comment:
This probably needs to be `@storybook/addon-viewport/manager` as it says it's deprecated in the comments in `/register.js` file and just requires `/manager` anyway

Reply:
Updated, thanks.